app/helpers/application_helper.rb

```diff
@@ -56,4 +56,17 @@ def text(name)
     text = SiteText.where(name: name).first.try :text
     render_markdown text if text.present?
   end
+
+  def dashboard_path
+    # Just incase this method is called when a current user does not exist
```

Comment:
I'd disagree, actually: there's a prominent use case for students using the site but not having created a user.

I would just change the condition to be `if @current.user.try :staff?`